packages/cashier-v2/src/utils/utils.ts

```diff
@@ -5,3 +5,17 @@ export const isServerError = (error: unknown): error is TErrorTypes.TServerError
 
 export const capitalizeFirstLetter = (targetString: string) =>
     targetString && targetString[0].toUpperCase() + targetString.slice(1);
+
+export const shuffleArray = <T>(array: T[] = []): T[] => {
+    const firstDigit = (num: number) => Number(String(num)[0]);
+
+    for (let i = array.length - 1; i > 0; i--) {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const crypto = window.crypto || (window as any).msCrypto; // to make it working in MS Explorer
```

Comment:
Hi @heorhi-deriv,
Do we still have clients using Microsoft Explorer?

Reply:
@aum-deriv 
believe that any user can use Microsoft Explorer to open our app.
the question is so we support Microsoft Explorer - the answer is no.
this util was copied from `shared` package and will be provided by @deriv-com/utils I believe so i decided to leave it as it is